src/libs/utils.js

```diff
@@ -74,6 +67,13 @@ const getAllChanges = async ({ rushChangePath, packagePaths, options = {} }) =>
       }
     }
   }
+  // Filter projects by version policy
+  if (options.versionPolicy && options.versionPolicy != "") {
```

Comment:
Is the second condition needed?

Reply:
For jest unit tests it required, because when we run unit test `options.versionPolicy` will be `undefined`

Reply:
`undefined` should also be picked by the first condition.

Reply:
Ah you are right, I got confused, Will update it.